tools/swift-inspect/Sources/swift-inspect/main.swift

```diff
@@ -17,6 +17,11 @@ import SwiftRemoteMirror
 internal struct UniversalOptions: ParsableArguments {
   @Argument(help: "The pid or partial name of the target process")
   var nameOrPid: String
+
+#if os(iOS) || os(macOS) || os(tvOS) || os(watchOS)
+  @Flag(help: "Fork a corpse of the target process")
```

Comment:
I really wish that we had a better description of this.  I suspect that for most this will not help particularly helpful.

Reply:
I added a longer description that's hopefully a little more clear.